ts/Maps/MapNavigation.ts

```diff
@@ -246,18 +247,45 @@ MapNavigation.prototype.update = function (
                 const unbind = addEvent(chart, 'load', (): void => {
                     // #15406: Make sure button hasnt been destroyed
                     if (button.element) {
+
+                        const expOptions = chart.options.exporting,
+                            expBtn = (
+                                expOptions &&
+                                expOptions.buttons &&
+                                expOptions.buttons.contextButton
+                            );
+
+                        // If the nav buttons are on the same side
+                        if (
+                            expBtn && defined(buttonOptions.y) &&
+                            chart.exportingGroup && chart.options &&
+                            chart.title && chart.title.textStr === '' &&
+                            chart.subtitle && chart.subtitle.textStr === '' &&
+                            (
+                                (
+                                    buttonOptions.align === 'right' &&
+                                    !expBtn.align
+                                ) ||
+                                (expBtn.align === buttonOptions.align)
+                            )
+                        ) {
```

Comment:
I think this is a place where we could think about simplifying this `if` statement.
What about putting some of these conditions to a variable and using only a variable (or a few variables) in the `if` statement?